posthog/celery.py

```diff
@@ -107,36 +107,38 @@ def setup_periodic_tasks(sender: Celery, **kwargs):
     sender.add_periodic_task(60.0, monitoring_check_clickhouse_schema_drift.s(), name="Monitor ClickHouse schema drift")
 
     if not settings.DEBUG:
-        sender.add_periodic_task(1.0, redis_celery_queue_depth.s(), name="1 sec queue probe", priority=0)
+        sender.add_periodic_task(10.0, redis_celery_queue_depth.s(), name="10 sec queue probe", priority=0)
     # Heartbeat every 10sec to make sure the worker is alive
     sender.add_periodic_task(10.0, redis_heartbeat.s(), name="10 sec heartbeat", priority=0)
 
     # Update events table partitions twice a week
     sender.add_periodic_task(
-        crontab(day_of_week="mon,fri", hour=0, minute=0), update_event_partitions.s()  # check twice a week
+        crontab(day_of_week="mon,fri", hour="0", minute="0"), update_event_partitions.s()  # check twice a week
     )
 
     # Send all instance usage to the Billing service
-    sender.add_periodic_task(crontab(hour=1, minute=0), send_org_usage_reports.s(), name="send instance usage report")
+    sender.add_periodic_task(
+        crontab(hour="1", minute="0"), send_org_usage_reports.s(), name="send instance usage report"
+    )
     # Update local usage info for rate limiting purposes - offset by 30 minutes to not clash with the above
-    sender.add_periodic_task(crontab(hour="*", minute=30), update_quota_limiting.s(), name="update quota limiting")
+    sender.add_periodic_task(crontab(hour="*", minute="30"), update_quota_limiting.s(), name="update quota limiting")
 
     # PostHog Cloud cron jobs
     # NOTE: We can't use is_cloud here as some Django elements aren't loaded yet. We check in the task execution instead
     # Verify that persons data is in sync every day at 4 AM UTC
-    sender.add_periodic_task(crontab(hour=4, minute=0), verify_persons_data_in_sync.s())
+    sender.add_periodic_task(crontab(hour="4", minute="0"), verify_persons_data_in_sync.s())
```

Comment:
That is wild that it takes strings 🤔

Reply:
I guess cron is strings so they're being very literal